Packages/com.unity.inputsystem/InputSystem/Editor/ControlPicker/InputControlPathEditor.cs

```diff
@@ -109,7 +109,19 @@ public void OnGUI(Rect rect, GUIContent label = null, SerializedProperty propert
             editButtonRect.width = 20;
             editButtonRect.height = 15;
 
-            var path = serializedProperty.stringValue;
+            var path = String.Empty;
+            try
+            {
+                path = serializedProperty.stringValue;
+            }
+            catch
+            {
+                // This try-catch block is a temporary fix for: https://jira.unity3d.com/browse/ISX-1436
```

Comment:
LGTM if this is a known issue in UITK/IMGUI integration. I take back what I said when we discussed this as in: Instead of providing an internal URL here linking to a page that is Unity internal, remove this URL (you may keep ISX-1436 reference) and instead link to this code from that issue in JIRA. Wouldn't that make more sense? I have now made the necessary linking in JIRA and made it a sub-task of that work to revert this workaround.